packages/fxa-settings/package.json

```diff
@@ -44,7 +44,12 @@
       }
     },
     "transform": {
```

Comment:
Maybe not relevant. I think this is what tripped up the build by the way... some sort of auto formatting appears to be takin place inside the docker container, and then we get a conflict here. I suspect husky is at fault. This was fixed recently, so a rebase on main should fix the issue.